src/main/java/org/apache/commons/net/util/NetConstants.java

```diff
@@ -23,7 +23,7 @@
  *
  * @since 3.8.0
  */
-public class NetConstants {
+public final class NetConstants {
```

Comment:
@jkbkupczyk 
You can't make this class final as it will break binary compatibility (it is public).